frontend/src/lib/components/SelectGradientOverflow.tsx

```diff
@@ -89,6 +91,7 @@ export function SelectGradientOverflow({
     const onBlur = (): void => {
         if (isOpen) {
             setOpen(false)
+            handleBlur?.()
```

Comment:
nit but prefer this to have a default value of `()=>{}` rather than `nullish ?` syntax